elastic/logs/tasks/index-setup.json

```diff
@@ -56,6 +56,55 @@
     "operation-type": "create-data-stream"
   }
 },
+{
+  "name": "validate-package-template-installation",
+  "operation": {
+    "operation-type": "validate-package-assets",
+    "packages": {{ p_integration_ratios.keys() | reject("==", "application-logs") | list | tojson }},
```

Comment:
Once https://github.com/elastic/rally-tracks/pull/335 is merged, we can remove the `reject` call.